frontend/cypress/integration/feature/feature.spec.ts

```diff
@@ -40,26 +39,20 @@ describe('feature', () => {
         );
     });
 
-    it('can add, update and delete a gradual rollout strategy to the development environment', async () => {
+    it('can add, update and delete a gradual rollout strategy to the development environment', () => {
         cy.addFlexibleRolloutStrategyToFeature_UI({
             featureToggleName,
-        }).then(value => {
-            strategyId = value;
-            cy.updateFlexibleRolloutStrategy_UI(
-                featureToggleName,
-                strategyId
-            ).then(() =>
-                cy.deleteFeatureStrategy_UI(featureToggleName, strategyId)
+        }).then(() => {
```

Comment:
The logic where we try to wait for the returned yielded `value` didn't really work, as it came back as `undefined` every time.